lib/core/collections.toit

```diff
@@ -845,12 +845,8 @@ abstract class Array_ extends List:
   // Optimized helper method for iterating over the array elements.
   abstract do_ end/int [block] -> none
 
-  /** Create a new array, copying up to old_length elements from this array. */
-  resize_for_list_ old_length/int new_length/int -> Array_:
-    result := Array_ new_length
-    for i := 0; i < old_length and i < new_length; i++:
-      result[i] = this[i]
-    return result
+  /** Create a new array, copying up to $copy_length elements from this array. */
```

Comment:
Use /// for consistency?

Also why do we call this copy_length and new_length here and not copy_size and new_size?